puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_develop.yaml

```diff
@@ -23,6 +23,7 @@
           - 1.9.3
           - 2.0.0
           - 2.1
+          - 2.2.2
```

Comment:
We define aliases in slave/manifests/init.pp, so you should be able to use "2.2" to mean whatever 2.2 release we aliased (currently 2.2.1).